installer/pkg/workflow/init.go

```diff
@@ -17,7 +17,7 @@ import (
 const (
 	generatedPath              = "generated"
 	kcoConfigFileName          = "kco-config.yaml"
-	tncoConfigFileName         = "tnco-config.yaml"
+	mcsoConfigFileName         = "mcso-config.yaml"
```

Comment:
drop this line, there is not `mcso-config.yaml`; missed this in https://github.com/openshift/installer/pull/232